adoc/admin-security-configure-authentication-connector.adoc

```diff
@@ -0,0 +1,344 @@
+== Configuring the Authentication Connector
+
+[IMPORTANT]
+====
+Before any add-on upgrade, please backup any runtime configuration changes, then restore the modification after upgrading.
```

Comment:
Do we still need this note after kustomize introduced? (but it's still good to have a note to backup in case)

Reply:
IIRC, we addressed this is because the dex connector setting would be reset to default after add-on upgrade, and also the custom OIDC certificate would be reset to use Kubernetes CA.

I remember these two issues both addressed in `4.2.2`, I'll remove it.